src/libs/antares/series/series.cpp

```diff
@@ -67,6 +67,10 @@ int TimeSeries::saveToFolder(const AreaName& areaID,
     return timeSeries.saveToCSVFile(buffer, 0);
 }
 
+int TimeSeries::saveToFile(const std::string& filename, bool saveEvenIfAllZero) const
+{
+    return timeSeries.saveToCSVFile(filename, 6, false, saveEvenIfAllZero);
```

Comment:
Why 6 ?

Reply:
It was the previous default precision used in the function to save the links